commit 58a3d790016bbe22e3b67d96a475a820306c67dd
Author: Jens Kraemer <jk@jkraemer.net>
Date:   Fri Feb 9 13:33:04 2018 +0800

    members extension: be more tolerant for participatory_spaces without model

diff --git a/decidim-core/app/presenters/decidim/home_stats_presenter.rb b/decidim-core/app/presenters/decidim/home_stats_presenter.rb
index 1e89f5f..8d44050 100644
--- a/decidim-core/app/presenters/decidim/home_stats_presenter.rb
+++ b/decidim-core/app/presenters/decidim/home_stats_presenter.rb
@@ -76,7 +76,7 @@ module Decidim
 
     def public_participatory_spaces
       @public_participatory_spaces ||= Decidim.participatory_space_manifests.flat_map do |manifest|
-        manifest.participatory_spaces.call(organization).public_spaces
+        manifest.participatory_spaces&.call(organization)&.public_spaces || []
       end
     end
 
